web/src/compose_tooltips.js

```diff
@@ -133,7 +134,9 @@ export function initialize() {
         // in the new design
         target: [".disabled-message-send-controls"],
         maxWidth: 350,
-        content: () => compose_recipient.get_posting_policy_error_message(),
+        content: () =>
+            compose_recipient.get_posting_policy_error_message() ||
+            compose_validate.get_disabled_send_tooltip(),
```

Comment:
@amanagr can confirm, but I have a suspicion that this doesn't live-update properly if you change the state within a single browser session; or at least I seem to recall problems with Tippy caching the return value of this function after the inputs change.

Reply:
I tested some more and tippy seems to work just fine, the tooltips are always what I'd expect even when the reasons for disabling change rapidly in a single session.

The problem (unrelated to any change in this PR) is that the state of the compose box does not immediately update on changing permissions, so it results, for example in the following right after closing the settings modal post changing enabling DMs:
![image](https://github.com/zulip/zulip/assets/68962290/26a22f58-ecb3-4226-9379-eacbf486c28e)

Reply:
OK, let's file a follow-up issue for that.

Reply:
Filed #27774